addons/max-pod/max-pod.addon

```diff
@@ -0,0 +1,11 @@
+# Name: max-pod
+# Description: Add-on to update the maximum pods value
```

Comment:
Could you add value as well here in the description so that it is available in the verbose view.

Reply:
I could, but the user is supposed to pass that value, the add-on is to allow the user to change the 'max-pods' value, not set it to a fixed 20. maybe we should just remove the `Default-Vars` ? WDYT?

Reply:
Then, updated description as 
```
Add-on to update the maximum pods value (default to 20)
```

Reply:
Also, need documentation update.